src/components/Party/Drawer/ActivityDrawerWidget.vue

```diff
@@ -0,0 +1,231 @@
+                        <div class="stack">
+                            <div class="actionLine">
+                                <!-- <img class="playerAvatar" :src="`/players/${player.icon}.png`" :alt="`${player.icon}.png`"> -->
+                                <!-- v-if="action.userId === user._id"    <label ><small>(You)</small></label> -->
+                                <div class="actionType"><strong>{{ actionSentence(action.actionType) }}</strong><br/>
+                                <label ><small>il y a {{ timeSince(action.date) }}</small></label></div>
+                            </div>
+                        </div>
+                    </div>
+            </div><label id="rightDrawerPull" class="drawerPull" for="rightDrawerCheck"></label>
+        </div>
+    </div>
+</template>
+
+<script>
+import {mapGetters} from 'vuex'
+
+export default {
+  name: "StatDrawerWidget",
+  props: {
+    actions: { type: Array },
+  },
+  methods: {
+    timeSince(date) {
+        if (typeof date !== 'object') {
+            date = new Date(date);
+        }
+
+        var seconds = Math.floor((new Date() - date) / 1000);
+        var intervalType;
+
+        var interval = Math.floor(seconds / 31536000);
+        if (interval >= 1) {
+            intervalType = 'an';
+        } else {
+            interval = Math.floor(seconds / 2592000);
+            if (interval >= 1) {
+                intervalType = 'mois';
+                return interval = ' ' + intervalType
+            } else {
+                interval = Math.floor(seconds / 86400);
+                if (interval >= 1) {
+                    intervalType = 'jour';
+                } else {
+                    interval = Math.floor(seconds / 3600);
+                    if (interval >= 1) {
+                        intervalType = "heure";
+                    } else {
+                        interval = Math.floor(seconds / 60);
+                        if (interval >= 1) {
+                            intervalType = "minute";
+                        } else {
+                            interval = seconds;
+                            intervalType = "seconde";
+                        }
+                    }
+                }
+            }
+        }
+
+        if (interval > 1 || interval === 0) {
+            intervalType += 's';
+        }
+
+        return interval + ' ' + intervalType;
+    },
+    actionSentence(actionType) {
+        switch (actionType) {
+            case "MOVE":
+                return "Déplacement"
+            case "POP":
+                return "Apparition"
+            case "NEXT_ROUND":
+                return "Round enregistré"
+            case "ATTACK":
+                return "Attaque"
+            default:
+                break;
+        }
+        return actionType;
+    }
+  },
+  computed: {
+    ...mapGetters(['user']),
+  }
+}
+</script>
+
+<style scoped>
```

Comment:
Maybe fix indentation in CSS

Reply:
already like that in all the vue components